server/jetstream_chaos_helpers_test.go

```diff
@@ -11,6 +11,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//go:build !skip_js_tests && !skip_js_cluster_tests && !skip_js_chaos_tests
```

Comment:
Would replace all that with `js_chaos_tests` and have the same tag in the non helper file. Then, you would need to probably add to the staticcheck command that tag so that it can check that too: `staticcheck -tags js_chaos_tests $GO_LIST`. However, if we don't include that one, then it would complain that some of the helpers you added in other files are not used (since chaos tests would not be compiled), such as `func (*cluster).waitOnClusterHealthz is unused (U1000)`, etc...